ui-tests/src/test/java/org/openmrs/reference/page/ManageFormsPage.java

```diff
@@ -117,7 +117,7 @@ public void clickOnCurrentDate() {
 
     @Override
     public String getPageUrl() {
-        return "/formentryapp/forms/extension.page";
+        return "/openmrs/formentryapp/forms.page";
```

Comment:
please remove /openmrs from the path. instead of `/formentryapp/forms/extension.page` just have `/formentryapp/forms.page`

Reply:
Hello @k-joseph  this is "/openmrs/formentryapp/forms.page", collected it on line **120**